### PR TITLE
Add responsive heading components

### LINKED ISSUE #3: Create "Heading" component
Create the Heading component:

- Base font-weight -> 400
- Available-sizes:
  - xs 
    - MOB: font size 20px  & line height 26px
    - TAB: font size: 22px & line height 28px 
  - l 
    - MOB: font size 24px & line height 28px 
    - TAB: font size 28px line height 36px
    - DT: font size 34px & line height 44px
- Allowed tags: h1, h2, h3, h4, h5, h6

### PR DESCRIPTION
**Pull request by rxcva**
I created the responsive heading componets.
 https://github.com/vbetivu-ellation/crunchygram/issues/3#event-5435096718

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,13 +3,30 @@ import styles from "./App.module.css";
 import { Text } from "./components/common";
 import StoreProvider from "./store";
 import Counter from "./components/Counter";
+import { HeadingL } from "./components/HeadingL/HeadingL.jsx"
+import { HeadingXL } from './components/HeadingXL/HeadingXL.jsx'
 
 
 function App() {
```
Comment (vbetivu-ellation) [inline src/App.js]: Let's revert the `App.js` component to not create conflicts.

**Review by vbetivu-ellation — COMMENTED** [review]
~1. It seems that you have troubles with `Prettier`. The files are not formatted properly.
2. The `Heading` component must be a single component (like the `Text` component).
3. It should receive the size ("xs", "l") through size props (like the `Text` component).
4. It should return a single tag that is configurable via props and can be one of (h1, h2, h3, h4, h5, h6).(see `Text` component's `as` prop)
5. As the component is a primary building block, it should not contain any extra logic that will prevent it from reusing (it should not have any other styling besides the font size and line height).
6. The tablet view starts at "568px" and the desktop view starts at "800px". (see `index.css` for reference)
7. We use `.js` files extension. Please update the `.jsx` extensions to match the project ones.~

**Comment by vbetivu-ellation** [comment]
Please reach out in slack if you need any help on the above comments.

File: src/App.js (hunk 1 above)
Comment (rxcva) [reply src/App.js]: Done

**Review by vbetivu-ellation — DISMISSED** [review]

**Review by darsenii-crunchyroll — APPROVED** [review]

**Pull request merged**